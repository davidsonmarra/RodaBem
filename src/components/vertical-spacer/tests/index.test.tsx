import React from 'react';
import {create} from 'react-test-renderer';
import VerticalSpacer from '..';

let mockHeight = 10;

const createComponent = () => create(<VerticalSpacer height={mockHeight} />);

describe('VerticalSpacer', () => {
  beforeEach(() => {
    mockHeight = 10;
  });

  it('renders correctly', () => {
    const tree = createComponent().toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders with height correctly', () => {
    const instance = createComponent().root;
    expect(instance.props.height).toBe(mockHeight);
  });
});
