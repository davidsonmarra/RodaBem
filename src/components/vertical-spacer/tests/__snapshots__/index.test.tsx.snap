// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerticalSpacer renders correctly 1`] = `
<View
  style={
    {
      "height": 10,
    }
  }
/>
`;
